vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -369,6 +370,8 @@ void BatContribution::StartReconcile(
   reconcile.category_ = category;
 
   ledger_->AddReconcile(viewing_id, reconcile);
+  ledger_->BroadcastReconcile(true);
```

Comment:
this is the kind of thing we really want to avoid moving forward. These methods have way too many side effects and I think we need to move these to more of an observer model. cc @tmancey

Reply:
Moved out of native ledger and his handled in rewards service through `OnReconcileStarted` and `OnReconcileComplete`